src/components/select/template.njk

```diff
@@ -0,0 +1,19 @@
+{% from "label/macro.njk" import govukLabel %}
+{% set labelArguments = {
+  text: params.label.text,
+  html: params.label.html,
+  hintText: params.label.hintText,
+  hintHtml: params.label.hintHtml,
+  for: params.id,
+  errorMessage: params.errorMessage
+} %}
+{%- if params.label %}
+{{- govukLabel(labelArguments) -}}
+{% endif -%}
+
+<select class="govuk-c-select
+  {%- if params.classes %} {{ params.classes }}{% endif %}{%- if params.errorMessage %} govuk-c-input--error{% endif %}" id="{{ params.id }}" name="{{ params.name }}"{% for attribute, value in params.attributes %} {{ attribute }}="{{ value }}"{% endfor %}>
+{% for item in params.items %}
+  <option value="{{ item.value }}"{%- if item.selected %} selected{% endif %} {%- if item.disabled %} disabled{% endif %}>{{ item.text }}</option>
```

Comment:
These could be e.g. `{{ ' selected' if item.selected }}` which IMO is cleaner – but less fussed about changing this.